document module deprecation process (#225)
voxpupuli/voxpupuli.github.io: _docs/deprecated_and_archived_modules.md

If you come across an archived Vox Pupuli project that you are still using or think should not have been archived, [let us know][email].
The decision to archive a module isn't necessarily final and can be undone.

### Deprecation process

> * The README.md needs a bolt message at the beginning

Reviewer: ```suggestion
* The README.md needs a bold message at the beginning
```

If you come across an archived Vox Pupuli project that you are still using or think should not have been archived, [let us know][email].
The decision to archive a module isn't necessarily final and can be undone.

### Deprecation process

> * An issue needs to be raised on the module to discuss wether it should be archived or not. The decision will be made by lazy consensus (as described in our governance guidelines)

Reviewer: ```suggestion
* An issue needs to be raised on the module to discuss whether it should be archived or not. The decision will be made by lazy consensus (as described in our governance guidelines)
```

> * An issue needs to be raised on the module to discuss wether it should be archived or not. The decission will be made by lacy consensus (as described in our governance guidelines)

Reviewer: ```suggestion
* An issue needs to be raised on the module to discuss whether it should be archived or not. The decision will be made by lazy consensus (as described in our governance guidelines)
```

* The README.md needs a bold message at the beginning

>   * It needs to mention that the module is archived

Reviewer: It'd suggest to use must/should per [RFC 2119](https://tools.ietf.org/html/rfc2119)
```suggestion
  * It must mention that the module is archived
```

  * If a successor exists, that one should be linked to

>   * The deprecation issue in that project needs to be linked

Reviewer: ```suggestion
  * The deprecation issue in that project must be linked
```
